march_hardware_interface/include/march_hardware_interface/march_hardware_interface.h

```diff
@@ -84,6 +84,7 @@ class MarchHardwareInterface : public hardware_interface::RobotHW
 
   /* Exponential smoothing constant of the velocity */
   static constexpr double ALPHA = 0.2;
+  static constexpr double MAX_EFFORT_CHANGE = 5000;
```

Comment:
Perhaps add a comment to describe what this constant means? Without having to read the code where it is used